adyenv6core/resources/adyenv6core-beans.xml

```diff
@@ -59,6 +60,7 @@
         <property name="adyenSelectedReference" type="String" />
         <property name="adyenSocialSecurityNumber" type="String" />
         <property name="adyenFirstName" type="String" />
+        <property name="installments" type="Integer" />
```

Comment:
shall we be consistent here with the rest and call it as well adyenInstallments ?

Reply:
It is consistent with "encrypteCardNumber" in same bean. Actually using "adyen" is not appropriate because this is the name of field which user sends in request.

Reply:
ok so it was a mistake that we prefix adyen on adyenPaymentMethod, etc.. make sense to keep this simple.